src/fit-bounds.js

```diff
@@ -3,10 +3,13 @@
 import Transform from 'mapbox-gl/js/geo/transform';
 import {LngLatBounds, Point} from 'mapbox-gl';
 
-export default function fitBounds(width, height, _bounds, options) {
-  const bounds = new LngLatBounds([_bounds[0].reverse(), _bounds[1].reverse()]);
-  options = options || {};
-  const padding = typeof options.padding === 'undefined' ? 0 : options.padding;
+export default function fitBounds(width, height, _bounds, {
+  padding = 0
```

Comment:
@vicapow - Good catch - I assume you are referring to the dropping of the `reverse` calls. Yes that was not intentional. 

Since there was no JSDoc it took me a while to figure out that this function only works on arrays and not Mapbox.LngLats, which is what I initially assumed, and the reason I removed `reverse` (which only works on arrays).

I'll make a new PR to fix this.